client-src/elements/chromedash-app.js

```diff
@@ -102,6 +102,7 @@ class ChromedashApp extends LitElement {
       pageComponent: {attribute: false},
       contextLink: {type: String}, // used for the back button in the feature page
       sidebarHidden: {type: Boolean},
+      selectedGate: {type: Number},
```

Comment:
Since it is the ID rather than a gate JSON object, I think this name would be more clear as `selectedGateId`.